Preview.md

```diff
@@ -0,0 +1,3 @@
+# Power Platform Tools [PREVIEW]
+
```

Comment:
Should this be named: README.Preview.md?
It's clear this file applies to the preview build only. but imagine if 'Preview' wasn't there, what would it be called?

Reply:
renamed